certbot/certbot/_internal/plugins/webroot.py

```diff
@@ -192,6 +192,24 @@ def _create_challenge_dirs(self):
             finally:
                 filesystem.umask(old_umask)
 
+            # On Windows, generate a local web.config file that allows IIS to serve expose
+            # challenge files despite the fact they do not have a file extension.
+            if not filesystem.POSIX_MODE:
+                web_config_path = os.path.join(self.full_roots[name], "web.config")
+                if os.path.exists(web_config_path):
+                    os.remove(web_config_path)
```

Comment:
I think Certbot on principle should be respectful of pre-existing files. To suggest a (admittedly contrived) potential problem with this: maybe the user has some other Windows client (Certify the Web, win-acme, whatever) which relies on their manually-set-up `web.config` file. Or maybe they have written the file in a specific way, with additional directives: such as exclusion from HTTP authorization.

Same feeling with cleanup; only removing the file if it has a known digest, similar to what Certbot does with the nginx/Apache options file.